llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -41,6 +40,9 @@ constexpr StringLiteral PaddingRef::KindString;
 #define MCFRAGMENT_NODE_REF(MCFragmentName, MCEnumName, MCEnumIdentifier)      \
   constexpr StringLiteral MCFragmentName##Ref::KindString;
 #include "llvm/MC/CAS/MCCASObjectV1.def"
+constexpr StringLiteral DebugInfoSectionRef::KindString;
+
+#define DWARF32_HEADER_SIZE 11
```

Comment:
In general: `#define` -> `constexpr`

Reply:
The header size will also depend on whether it is a TU or CU, but  I think here you only mean "number of bytes up to and including the `length` field of the header", right? Since that's all we care about